src/manage_sql_nvts.c

```diff
@@ -1477,13 +1476,15 @@ update_nvts_from_vts (entity_t *get_vts_response,
 
       if (strcmp (osp_vt_hash, db_vts_hash ? db_vts_hash : ""))
         {
-          g_warning ("%s: SHA-256 hash of the VTs in the database does not"
-                     " match the one from the scanner.",
-                     __func__);
-          g_debug ("%s: Hash from OSP: %s, from database: %s",
-                   __func__, osp_vt_hash, db_vts_hash);
+          g_warning ("%s: SHA-256 hash of the VTs in the database (%s)"
+                     " does not match the one from the scanner (%s).",
+                     __func__, osp_vt_hash, db_vts_hash);
```

Comment:
Thanks.  The hash args should be swapped.

Reply:
I fixed this in 99cbf9483b36a6da684a66a729c0914ba8b1b531